core/listener.go

```diff
@@ -57,7 +57,7 @@ func NewListener(
 		hashBroadcaster:   hashBroadcaster,
 		construct:         construct,
 		store:             store,
-		listenerTimeout:   2 * blocktime,
+		listenerTimeout:   3 * blocktime,
```

Comment:
Can we reliably track intentionally skipped blocks to prevent false positive triggers?

Reply:
@walldiss it's not that there's block skipping, it's more that there can be a delay every so often. I wouldn't say that this is something we need to keep track of at the moment.

I think if we have a block time of 10s, it would be rare to hit a 30s block time @Wondertan so I don't know if increasing the buffer that much is worth it.

Reply:
i'm happy to increase it though - it's not dangerous on either side (whether we are too quick to re-trigger or whether we wait a bit) as we have syncer to help clean up any gaps.

Reply:
Great, let's increase then